Model/Sales/MagentoOrderCollection.php

```diff
@@ -180,6 +180,10 @@ public function setMyParcelTrack()
 
         $this->myParcelCollection = $newCollection;
 
+        if ($this->options['return_in_the_box']) {
```

Comment:
Is return_in_the_box altijd aanwezig als key? Anders `$this->options['return_in_the_box'] ?? false` doen of `! empty($this->options['return_in_the_box'])`

Reply:
`$this->options['return_in_the_box']` wordt op regel 94 standaard op false gezet!

Reply:
oja, dat is waar